### ISSUE TITLE
2D building placement

### ISSUE THREAD

**Issue by cantorek** [suriyun-production/mmorpg-kit-docs#966]
When placing 2D buildings via mouse, the building won't show up under mouse pointer, so it is impossible to place it in desired location.

tempVector3 should be assigned to ConstructingBuildingEntity.Position. It's assigned for 3D but not for 2D.

Fix:
line 68 of UnityMultiplayerARPG\Core\Scripts\Gameplay\CharacterControllerSystems\Default\PlayerCharacterController_FindObjects.cs
```
                } else if (CurrentGameInstance.DimensionType == DimensionType.Dimension2D)
                {
                    ConstructingBuildingEntity.Position = tempVector3;
                }
```

**Comment by insthync** [comment]
Thank you :)

**Comment by insthync** [comment]
Fixed in 1.65e
